spec.html

```diff
@@ -7493,7 +7493,7 @@ <h1>GetGlobalObject ( )</h1>
 
   <emu-clause id="sec-jobs-and-job-queues">
     <h1>Jobs and Job Queues</h1>
-    <p>A Job is an abstract operation that initiates an ECMAScript computation when no other ECMAScript computation is currently in progress. A Job abstract operation may be defined to accept an arbitrary set of job parameters.</p>
+    <p>A <dfn id="job">Job</dfn> is an abstract operation that initiates an ECMAScript computation when no other ECMAScript computation is currently in progress. A Job abstract operation may be defined to accept an arbitrary set of job parameters.</p>
```

Comment:
@ljharb Accidentally included the Job and Job Queue dfn editorial commit as part of this PR?

Reply:
No, it was intentional. I can certainly split it out if you prefer, though.

Reply:
Nope, just making sure it wasn't accidental.